### ISSUE TITLE
When both fixed and includeSpinner input options are set to false loading-bar is hidden

### ISSUE THREAD

**Issue by karthilxg** [aitboudad/ngx-loading-bar#85]
When setting the `fixed` and `includeSpinner` input properties to `false` the loading-bar is not visible irrespective of `includeBar` input is set to `true`.

Check this demo: https://stackblitz.com/edit/angular-nvgyoa

```ts
<ngx-loading-bar [includeSpinner]="false" [fixed]="false" [includeBar]="true"></ngx-loading-bar>
```

**Comment by aitboudad** [comment]
fixed in `2.1.0`

**Comment by karthilxg** [comment]
The latest version fixes the issue. 👍
